chaos-bugbounty-list.json

```diff
@@ -1844,6 +1852,13 @@
         "t-systems.com"
       ]
     },
+    {
+      "name": "Dexalot",
+      "url": "https://hackenproof.com/dexalot/dexalot-smart-contracts",
+      "bounty": true,
+      "domains": [
+      ]
```

Comment:
@ehsandeep is this a valid scope - I checked their bounty page and they don't in fact list any web properties...but do we want to include non-web programs here?

Reply:
It would be more useful to have a list of public programs with their associated contact information, regardless of their scope. Therefore, we should include all public programs.